assets/agenda/components/AgendaCoverages.tsx

```diff
@@ -45,63 +48,89 @@ function AgendaCoveragesComponent({item, coverages, wireItems, actions, user, on
                     ))}
                 </div>
             )}
-            {coveragesWithState.map((coverage: any) => (
-                <div
-                    className={classNames(
-                        'coverage-item',
-                        {'coverage-item--clickable': onClick}
-                    )}
-                    key={coverage.coverage_id}
-                    onClick={onClick}
-                    title={onClick ? gettext('Open {{agenda}} in a new tab', window.sectionNames) : onClick}
-                >
+            {coveragesWithState.map((coverage: ICoverage) => {
+                const assigneeName = getCoverageAsigneeName(coverage);
+                const deskName = getCoverageDeskName(coverage);
+                const assignedUserEmail = coverage.assigned_user_email;
+                const assignedDeskEmail = coverage.assigned_desk_email;
+                const subject = gettext(`Coverage inquiry from {{sitename}} user: ${item.name || item.slugline}`,{sitename: window.sitename});
```

Comment:
there should be also `{{item}}` in the string, otherwise it would be different gettext string for each item